apps/betterangels/package.json

```diff
@@ -79,7 +79,8 @@
     "expo-file-system": "*",
```

Comment:
Hi @Davit-BetterAngels 

I'm getting "Something went wrong" message when I tapped on the Team section.
So I couldn't test it. Let me know when this is fixed.

<img src="https://github.com/user-attachments/assets/79f1ff9b-cccb-4436-b8da-6ecc481c6655" width="320" />

Reply:
Nvm, I forgot to update the build. Now it has and I confirmed the Team picker. Yes, the UI needs to be updated but good for now.

Reply:
Hi @Davit-BetterAngels 

This is what i see in Android. 
Can you fix this as the same as iOS?

<img src="https://github.com/user-attachments/assets/8ac952aa-39b6-46a6-be98-56f904e5604d" width="320" />

Reply:
Hi @Davit-BetterAngels 

I am not sure this is a side effect from this PR, but please check the wheel/picker in the Household section.

On iOS, you can't enter the name, gender, and DOB after you select one household member e.g., Aunt or Mother
On Android, you can't select the household member. Even though you selected Grandparent, it goes back to Aunt. 

Let me know if this is something you need to look into in a different ticket. I can create one once I know this is not the side effect from this PR.

Reply:
fixed

Reply:
Hi @Davit-BetterAngels 

I think the UI is off unfortunately...

It should have the title "Team" and the team name next to it (wrap it if it's too long) as it's shown in the screenshot. Would you be able to fix the UI?

<img width="230" alt="image" src="https://github.com/user-attachments/assets/a2f591a3-710d-4565-ad97-d00bfa2e10a3">
<img width="230" alt="image" src="https://github.com/user-attachments/assets/8cd60644-e145-4f00-8dbd-f9c995d76355">

Let me know if you have any question.

Reply:
Btw, the side effect was cleared and now I can enter the household info. Thanks!